pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -431,27 +456,37 @@ func TestAddPodWillReplaceAssumed(t *testing.T) {
 	}}
 
 	for i, tt := range tests {
-		cache := newSchedulerCache(ttl, time.Second, nil)
-		for _, podToAssume := range tt.podsToAssume {
-			if err := assumeAndFinishBinding(cache, podToAssume, now); err != nil {
-				t.Fatalf("assumePod failed: %v", err)
-			}
-		}
-		for _, podToAdd := range tt.podsToAdd {
-			if err := cache.AddPod(podToAdd); err != nil {
-				t.Fatalf("AddPod failed: %v", err)
-			}
-		}
-		for _, podToUpdate := range tt.podsToUpdate {
-			if err := cache.UpdatePod(podToUpdate[0], podToUpdate[1]); err != nil {
-				t.Fatalf("UpdatePod failed: %v", err)
-			}
-		}
-		for nodeName, expected := range tt.wNodeInfo {
-			t.Log(nodeName)
-			n := cache.nodes[nodeName]
-			deepEqualWithoutGeneration(t, i, n, expected)
-		}
+		t.Run(fmt.Sprintf("%v", i), func(t *testing.T) {
```

Comment:
*sigh* If there was only one test case, maybe it didn't need a table.

Anyway, I don't think loop index is a very helpful subtest name. I'd say let's either figure out a useful name for this subtest, or delete the table above. We could convert the test case struct fields into local variables used below.

Reply:
yeah, when i came across this ( and i think there are others too ) it seemed like it should just be a normal test without a table.

Thinking that's the approach ill take here, and if/when there is a need for a table it can be added later.